users/danielo515/config.h

```diff
@@ -0,0 +1,4 @@
+#if defined(COMBO_ENABLE)
```

Comment:
This should have an include guard: 
```suggestion
#pragma once

#if defined(COMBO_ENABLE)
```